.github/workflows/release.yml

```diff
@@ -0,0 +1,319 @@
+name: Release
+run-name: Publish prerelease ${{ github.ref_name }} by @${{ github.actor }}
```

Comment:
This workflow is not only for prerelease, it also has a `upload-release` job

Reply:
Let's leave it as it is, the `upload-release` job won't be triggered in our case.

Reply:
Resolved in https://github.com/daeuniverse/daed/pull/65/commits/7f727fa1ca23c1ab84d4e980ddc1347a9681c538